### ISSUE TITLE
Generic webhook notification

### ISSUE THREAD

**Issue by geekscrapy** [containrrr/watchtower#1203]
Would love to see the generic webhook in shoutrrr to be usable in watchtower. My use case is to send each event to Home assistant which can then notify me if something is wrong. This means I can just use the Home assistant app notifications rather than having to setup something like gotify etc. just to receive notifications

**Comment by piksel** [comment]
This is not well documented, but [it *is* possible](https://containrrr.dev/shoutrrr/v0.5/services/generic/) using the latest version of shoutrrr (which was included in the just released watchtower v1.4.0).
There is no way to provide a custom template (like a JSON payload) for only one notification service though, so you have to either only use the home assistant webhooks and use a JSON template, or parse the content.
Basically, you replace the "https" in your webhook URL with "generic" and pass it to watchtower as a notification url:
```
--notification-url generic://ha.example.com/api/webhook/your-webhook-name
```
It should work pretty well with the new `--notification-report` templates, but I have yet to write a JSON template for it.

**Comment by geekscrapy** [comment]
Oh, awesome! That's good enough for me!
Many many thanks
